src/main/java/com/inventorysystem/inventory/VendorProfitController.java

```diff
@@ -49,10 +50,16 @@ public String addProduct(Product product){
         return "main";
     }
 
-     /** GetMapping for /product endpoint
+     /** GetMapping for /view-products endpoint
      * @return all products entered.
```

Comment:
As long as you're in the JavaDoc, move the first line of text to a new line.  We typically don't have any text on the /** line.